### ISSUE TITLE
null is not an object (evaluating 'el.paused') in iOS

### ISSUE THREAD

**Issue by cindyloo** [twilio/twilio-video.js#1863]
this error is in a callback in `localvideotrack.js` that is already trying to address a workaround
```
}).catch(function (error) {
            log.warn('Failed to detect silence and restart:', error);
})
.finally(function () {
            // If silent frames were not detected, then pause the dummy element again.
            el = localVideoTrack._dummyEl;  // <- fails if localVideoTrack is null (caught in error line above)
            if (!el.paused) {
                el.pause();
            }
```

the easiest fix would be for you to test localVideoTrack?._dummyTrack

**Software versions:**

 - [x] Browser(s): Safari
 - [x] Operating System: iOS 15
 - [x] twilio-video.js: 2.23.0
 - [x] Third-party libraries (e.g., Angular, React, etc.):React

**Comment by manjeshbhargav** [comment]
Hi @cindyloo ,

Can you share a screenshot of the error, and reproduction steps?

**Comment by cindyloo** [comment]
![IMG_1512](https://user-images.githubusercontent.com/659092/186507467-a8a0ca5c-335a-45df-a334-bdac0d274acc.PNG)

**Comment by cindyloo** [comment]
built on the basis of the twilio-react-app, 
start a session on desktop, send invite to ios. when user signs in, I get this error. This happens if the el.muted is set to true.

If I do something like this:
`if (isMobile){
el.muted =false
}`
then I am less likely to get this error

**Comment by edleatherbury** [comment]
We see many of these errors in our production logs. We're unable to reproduce locally, but it seems to only come from iOS devices. We're guessing it has something to do with how iOS backgrounds and pauses video, and how the SDK handles that. Agree with @cindyloo that a null check would fix the error.

This is from minified JS, but it's the same spot mentioned by @cindyloo 

`TypeError: null is not an object (evaluating '(y=p._dummyEl).paused')`

**Comment by manjeshbhargav** [comment]
@cindyloo , @edleatherbury ,

Thanks for the clarification. I will create an internal ticket and get this scheduled in our sprint. Please watch this space for updates.

**Comment by manjeshbhargav** [comment]
@cindyloo , @edleatherbury ,

Can you verify whether our current release candidate `npm install twilio/twilio-video.js#2.24.1-rc1` fixes this issue for you?

**Comment by edleatherbury** [comment]
@manjeshbhargav We've never reproduced the issue locally, we've only seen it in production logs. Therefore pulling down the RC probably won't be much help to you.
We will gladly verify the general release when it is available, however.

**Comment by cindyloo** [comment]
we will install that and report back soon

**Comment by cindyloo** [comment]
actually @manjeshbhargav - is that the right package? My terminal is just doing nothing trying to install that package
```
npm install twilio/twilio-video.js#2.24.1-rc1
```

**Comment by cindyloo** [comment]
let me know asap please , TY

**Comment by PikaJoyce** [comment]
Hi @cindyloo,

This issue has now been fixed on the [latest version 2.24.1](https://github.com/twilio/twilio-video.js/releases/tag/2.24.1). I'll now be closing this issue but please don't hesitate to let us know if you're still experiencing issues with this latest release.

Best,
Joyce

**Comment by cindyloo** [comment]
it appears to be fixed! hurray! I had an older version of node/npm so the # notation didn't work, nor was I familiar with it
